ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -1120,4 +1120,55 @@ ActorScope,string,Optional,ProcessEvent,,,
+EventSeverity,string,Mandatory,UserManagement,Enumerated,Informational|Low|Medium|High,
+EventSchema,string,Mandatory,UserManagement,,UserManagement,
+EventSchemaVersion,string,Mandatory,UserManagement,SchemaVersion,,
+ActingAppId,string,Optional,UserManagement,,,
+ActiveAppName,string,Optional,UserManagement,,,
+ActingAppType,string,Optional,UserManagement,Enumerated,Process|Browser|Resource|Other,
+HttpUserAgent,string,Optional,UserManagement,Useragent,,
+ActorUsername,string,Mandatory,UserManagement,Username,,
+ActorUserIdType,string,Conditional,UserManagement,Enumerated,SID|UID|AADID|OktaId|AWSId,ActorUserId
+ActorUserId,string,Optional,UserManagement,,,
+ActorSessionId,string,Optional,UserManagement,,,
+EventSubType,string,Optional,UserManagement,Enumerated,UserRead|UserCreated|GroupCreated|UserModified|GroupModified,
+GroupId,string,Optional,UserManagement,,,
+GroupIdType,string,Conditional,UserManagement,Enumerated,SID|UID,GroupId
+GroupName,string,Optional,UserManagement,,,
+GroupNameType,string,Conditional,UserManagement,Enumerated,UPN|Windows|DN|Simple,GroupName
+GroupType,string,Optional,UserManagement,Enumerated,Local Distribution|Local Security Enabled|Global Distribution|Global Security Enabled|Universal Distribution|Universal Security Enabled|Other,
+GroupOriginalType,string,Optional,UserManagement,,,
+SrcFQDN,string,Optional,UserManagement,FQDN,,
+SrcDvcId,string,Optional,UserManagement,,,
+SrcDvcScopeId,string,Optional,UserManagement,,,
+SrcDvcScope,string,Optional,UserManagement,,,
+SrcDvcIdType,string,Optional,UserManagement,Enumerated,AzureResourceId|MDEid,
+SrcDeviceType,string,Optional,UserManagement,Enumerated,Computer|Mobile Device|IOT Device|Other,
+SrcGeostring,string,Optional,UserManagement,,,
+SrcGeostring,string,Optional,UserManagement,,,
+SrcGeoCity,City,Optional,UserManagement,City,,
+SrcGeoLatitude,Latitude,Optional,UserManagement,,,
+SrcGeoLongitude,Longitude,Optional,UserManagement,,,
+TargetUserId,string,Optional,UserManagement,,,
+TargetUserIdType,string,Conditional,UserManagement,Enumerated,SID|UID|AADID|OktaId|AWSId,TargetUserId
+TargetUsername,string,Optional,UserManagement,Username,,
+TargetUsernameType,string,Conditional,UserManagement,Enumerated,UPN|Windows|DN|Simple,TargetUsername
+TargetUserType,string,Optional,UserManagement,Enumerated,Regular|Machine|Admin|System|Application|Service Principal|Other,
+TargetOriginalUserType,string,Optional,UserManagement,,,
+PreviousPropertyValue,string,Optional,UserManagement,,,
+NewPropertyValue,string,Optional,UserManagement,,,
+EventResultDetails,string,Recommended,UserManagement,Enumerated,NotAuthorized|Other,
+Src,string,Recommended,UserManagement,,,
+SrcIpAddr,IP address,Recommended,UserManagement,IP Address,,
+SrcHostname,string,Recommended,UserManagement,Hostname,,
+SrcDomain,string,Recommended,UserManagement,Domain,,
+SrcDomainType,string,Conditional,UserManagement,Enumerated,Windows|FQDN,SrcDomain
+ActorOriginalUserType,string,Optional,UserManagement,,,
```

Comment:
Fields like Dvc, EventCount, EventStartTime, EventEndTime, EventProduct, EventVendor, etc. are missing.

Reply:
Added all common fields.
I have created an automation script that will create a schema for ASimTester.csv that created the CSV per schema based on the documentation webpage.

There for some fields were shown as another Class because it was differently noted a the documentation page.

Thanks for your detailed reviewing of this PR.